assets/js/src/core/components/icon/icon.tsx

```diff
@@ -24,24 +24,43 @@ export interface IconProps {
 
 export const Icon = ({ name, options, className }: IconProps): React.JSX.Element => {
   const iconLibrary = useInjection<IconLibrary>(serviceIds.iconLibrary)
-  const SvgIcon = iconLibrary.get(name)
   const width = options?.width ?? 16
   const height = options?.height ?? 16
 
-  if (SvgIcon === undefined) {
-    return <div style={ { width, height } } />
+  const renderIcon = (): React.JSX.Element => {
+    const isFile = name.endsWith('.svg')
```

Comment:
We shouldn't do it like this but reflect the original icon type which comes from the server ;)